src/components/shared/TreeView.js

```diff
@@ -13,6 +13,8 @@ import ContextMenuTrigger from './ContextMenuTrigger';
 import type { IconWithClassName } from '../../types/reducers';
 import type { CssPixels } from '../../types/units';
 
+const PAGE_KEYS_DELTA = 15;
```

Comment:
Could you include a comment on how you came up with this magic number?

Reply:
I think using a magic number here is fine, but I think the more consistent behavior with other UI is to have it move based on the size of the viewport. That's more complicated to implement, so this is fine and provides value.

Reply:
That was also my rationale here :)

Reply:
added a comment